### PR TITLE
Another fix for DI/minification

### PR DESCRIPTION
**Pull request by corkupine**

### CODE REVIEW

**Comment by corkupine** [comment]
I will presume this will be tag 0.0.3.

File: angular-auth.js [hunk 1]
```diff
@@ -63,5 +63,5 @@ angular.module('angular-auth', [])
             };
 
         };
-        $httpProvider.responseInterceptors.push(interceptor);
+        $httpProvider.responseInterceptors.push(['$rootScope', '$q', 'requests401', interceptor]);
```
Comment (Iristyle) [inline angular-auth.js:66]: I'm curious... whats the explanation on this one?

Comment (corkupine) [inline angular-auth.js:66]: Same as the other - when registering things with Angular via .controller, .factory, .service, .config, .push - you need to specify the deps by name or uglification blows everything up.

Comment (Iristyle) [inline angular-auth.js:66]: The Angular array threw me off.  Dependency... dependency... dependency .... function.

I think we need to add a check to the build to look for these -- this is a total landmine.

**Pull request merged**